### PR TITLE
{tools}[foss/2021b] SIERRA v1.1.12 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by jfgrimm**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/SIERRA/SIERRA-1.1.12-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        'source_urls': ['https://github.com/swarm-robotics/sierra/archive'],
+        'checksums': ['71dc8a62f84191fceed9e90aea60860038f4ba7ed63f965aad8f1b4028533f73'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/sierra-cli'],
+    'dirs': [],
+}
+
```
Comment (casparvl) [inline easybuild/easyconfigs/s/SIERRA/SIERRA-1.1.12-foss-2021b.eb:97]: Is there a sensible `sanity_check_commands`  you can define? (e.g. a `--version`)

Comment (jfgrimm) [reply easybuild/easyconfigs/s/SIERRA/SIERRA-1.1.12-foss-2021b.eb:97]: no, as far as I understand you need an "experiment" to have been setup for the commands to output anything useful

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in total)
software1.lisa.surfsara.nl - Linux debian 10.12, x86_64, Intel(R) Xeon(R) Silver 4110 CPU @ 2.10GHz, Python 2.7.16
See https://gist.github.com/fbdbe2bf57a5dbde64a27d8c9b66410e for a full test report.

File: easybuild/easyconfigs/s/SIERRA/SIERRA-1.1.12-foss-2021b.eb (hunk 1 above)
Comment (casparvl) [reply easybuild/easyconfigs/s/SIERRA/SIERRA-1.1.12-foss-2021b.eb:97]: Ok

**Comment by casparvl** [comment]
Hm, failure on software1 is actually a problem in the sanity check of the `ARGoS` dependency, not a problem with _this_ PR. For the sake of this PR we can ignore it.

**Pull request closed without merging**